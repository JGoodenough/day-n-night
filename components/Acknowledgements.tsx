import { A } from '@expo/html-elements';
import { AppFontSizes } from '../constants/ui';
import { Text, StyleSheet, SafeAreaView } from 'react-native';

const Acknowledgements = () => {
  return (
    <SafeAreaView style={styles.Acknowledgements__Container}>
      <Text style={styles.Acknowledgements__Text}>
        Powered by:{' '}
        <A
          style={{ color: 'blue', textAlign: 'center' }}
          href="https://sunrisesunset.io/"
        >
          SunriseSunset.io
        </A>
      </Text>
    </SafeAreaView>
  );
};

export default Acknowledgements;

const styles = StyleSheet.create({
  Acknowledgements__Container: {
    width: '100%',
    height: '5%',
  },
  Acknowledgements__Text: {
    textAlign: 'center',
    fontFamily: 'WaitingfortheSunrise_400Regular',
    fontSize: AppFontSizes.MainFontSize,
  },
});
