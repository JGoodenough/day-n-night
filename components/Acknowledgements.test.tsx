import renderer from 'react-test-renderer';

import Acknowledgements from './Acknowledgements';
import { A } from '@expo/html-elements';

describe('<Acknowledgements />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Acknowledgements />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
