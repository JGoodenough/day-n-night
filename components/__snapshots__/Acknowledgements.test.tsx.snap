// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Acknowledgements /> renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "height": "5%",
      "width": "100%",
    }
  }
>
  <Text
    style={
      {
        "fontFamily": "WaitingfortheSunrise_400Regular",
        "fontSize": 18,
        "textAlign": "center",
      }
    }
  >
    Powered by:
     
    <Text
      accessibilityRole="link"
      onPress={[Function]}
      style={
        {
          "color": "blue",
          "textAlign": "center",
        }
      }
    >
      SunriseSunset.io
    </Text>
  </Text>
</RCTSafeAreaView>
`;
